File: cf-705a.py
n = int(input())
isHate = True
while n:
    n -= 1
    print("I", end = " ")
    if isHate:
        print("hate", end = " ")
    else:
        print("love", end = " ")

    if n > 0:
        print("that", end = " ")

    isHate = not isHate

print("it")

File: cf-705a.cpp
#include <iostream>
using namespace std;

int main () {
    int n;
    cin >> n;
    bool isHate = true;
    while (n--) {
        cout << "I ";
        if (isHate)
            cout << "hate";
        else
            cout << "love";
        if (n != 0)
            cout << " that ";

        isHate = !isHate;
    }
    cout << " it";
    return 0;
}
